src/sdk/account/utils/toSigner.ts

```diff
@@ -1,10 +1,12 @@
+import { Wallet } from "ethers"
```

Comment:
this means consumers need to pull in ethers code too? even though they may not even use ethers?

Reply:
Hi @jxom yep, that was the initial version of it but it's changed now, still looking into the best way to deal with this without relying on ethers package and without having to add a lot of code on our side. As of now, I found an issue but is not very elegant.